python/src/analytics_attributes.py

```diff
@@ -186,6 +195,24 @@ def uri_attributes(self, metrics_parameter, metrics_required):
 
         return attributes
 
+    def data_attributes(self, metrics_parameter, metrics_required):
+        """
+        Provides the attributes as a dict to provide for the
+        POST that requests or initiates the report.
+        """
+        self.verify_attributes()
+
+        attributes = {"start_date": self._start_date, "end_date": self._end_date}
+
+        metrics = self.metrics_array(required=metrics_required)
+        if metrics and len(metrics) > 0:
```

Comment:
The first test (for truthyness) implies the second, so only the first is needed.

Reply:
Thanks! This time, I was translating from nodejs to python and missed this nuance in the difference between JavaScript and python. :)